web/src/lib/stores.ts

```diff
@@ -5,6 +5,7 @@ export const token: Writable<string | null> = writable(null);
 
 export const user: Writable<User | null> = writable(null);
 
+export const store: Writable<User | null> = writable(null);
```

Comment:
We already have a user above, why have another user?

Reply:
For the user you are viewing and the user that is logged in - maybe there is another way to do this?